conditional/blueprints/member_management.py

```diff
@@ -39,6 +39,7 @@
 from conditional.util.ldap import ldap_is_onfloor
 from conditional.util.ldap import __ldap_add_member_to_group__ as ldap_add_member_to_group
 from conditional.util.ldap import __ldap_remove_member_from_group__ as ldap_remove_member_from_group
+from conditional.util.ldap import __ldap_is_member_of_group__ as ldap_is_member_of_group
```

Comment:
Just import them using the names with underscores, instead of casting them to a different local name.